core/impl/src/main/java/com/blazebit/persistence/impl/function/window/oragg/WindowOrAggFunction.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2014 - 2019 Blazebit.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.blazebit.persistence.impl.function.window.oragg;
+
+import com.blazebit.persistence.impl.function.window.AbstractWindowFunction;
+
+/**
+ * @author Jan-Willem Gmelig Meyling
+ * @author Sayra Ranjha
+ * @since 1.4.0
+ */
+public class WindowOrAggFunction extends AbstractWindowFunction {
```

Comment:
Because the standard one is an aggregrate function so it should have a different name so that the window function does not trigger for example implicit group bys to be rendered. I thought it would be the cleanest to keep aggregate and window functions separate, similar to how the implementations for sum, max, min, avg are already seperate (although they are defined in JPA alteady)

Reply:
Yeah, you are totally right. I missed that.